ch_pipeline/hfb/sidereal.py

```diff
@@ -0,0 +1,114 @@
+"""Regrid the data to sidereal time."""
+import numpy as np
```

Comment:
```suggestion
"""Regrid the data to sidereal time."""

import numpy as np
```

New version of `black` wants a blank line here.